### ISSUE TITLE
1.0.0 CLAW playbook shows wall of red, error, but it seems to work

### ISSUE THREAD

**Issue by mjordan** [Islandora/documentation#1144]
Just installed a shiny new 1.0.0 playbook :smile:. Saw a wall of red and this error:

```
Error: Call to a member function getSchema() on null in Drupal\\views\\EntityViewsData->mapFieldDefinition() (line 383 of /var/www/html/drupal/web/core/modules/views/src/EntityViewsData.php)."]}
...ignoring
```

But as far as I can tell, all views, contexts, etc. installed by the islandora_defaults module are intact. Not sure if this is new so I thought I'd report it.

**Comment by whikloj** [comment]
I believe the ansible line for that should specify that a failure is expected. https://github.com/Islandora-Devops/claw-playbook/blob/master/post-install.yml#L15

**Comment by mjordan** [comment]
@whikloj I can confirm that line is present when I built the VM.

**Comment by whikloj** [comment]
Then this error will go away with Drupal 8.7

**Comment by mjordan** [comment]
Ah. Sorry, I thought we were on 8.7. Should I close this?

**Comment by jonathangreen** [comment]
It might not be worth it since 8.7 it maybe around the corner. If we don't want the wall of scary red text, another option would be to use `failed_when` to make the task not fail. That way it won't be red and failed looking.

```yaml
  - name: Install islandora_defaults module (fail ok)
      command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
      failed_when: no
```

https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#controlling-what-defines-failure

**Comment by whikloj** [comment]
You can close it or leave it open. Might want to leave it open for now as we also have #873 and #885 that are closed tickets about this. So leaving one open might be a good idea, until it is closed forever.

**Comment by mjordan** [comment]
Ok, will do (nothing for now).

**Comment by Natkeeran** [comment]
I can confirm that this issue is resolved in 8.7.

**Comment by kstapelfeldt** [comment]
As this was confirmed, and is old, we're closing it.
